lib/email/receiver.rb

```diff
@@ -203,6 +205,20 @@ def is_bounce?
       true
     end
 
+    def is_from_reply_by_email_address?
+      reply_addresses = [SiteSetting.reply_by_email_address]
```

Comment:
Why not use the `reply_by_email_address_regex` function?

Reply:
reply_by_email_address_regex is too specific. I want to handle the case where the spammer has removed the + and the reply_key

Reply:
Any chances you could merge both methods and add an argument?